src/Interop/Android/AndroidCamera.cs

```diff
@@ -0,0 +1,38 @@
+﻿#if ANDROID
+using Android.Hardware.Camera2;
```

Comment:
We only need an OS specific target for android to get the camera ids, I just added those if we need them in any future updates and they can be removed